### PR TITLE
[zh-cn]: update translation of HTTP `Cross-Origin-Resource-Policy` header

### PR DESCRIPTION
**Pull request by T34-active**
### Description
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Resource-Policy
### Related issues and pull requests
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/http/headers/cross-origin-resource-policy/index.md
* Last commit: https://github.com/mdn/content/commit/b54373ab9025ceb6eb404bd2538ebd4c01576c60

### CODE REVIEW

File: files/zh-cn/web/http/headers/cross-origin-resource-policy/index.md [hunk 1]
```diff
@@ -1,32 +1,43 @@
...
 {{HTTPSidebar}}
 
-> **备注：** 由于[Chrome 浏览器中的一个 Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=952834), 设置 Cross-Origin-Resource-Policy（跨域资源策略）会使文件下载失败：当从设置了 CORP 请求头的资源服务器下载资源时，浏览器会阻止用户使用“保存”或“另存为”按钮将文件保存到本地。在决定生产环境中是否使用这一特性（CORP）之前需要慎重考虑。
-
-**`Cross-Origin-Resource-Policy`** 响应头会指示浏览器阻止对指定资源的无源跨域/跨站点请求。
-
-| Header type                           | {{Glossary("Response header")}} |
-| ------------------------------------- | ------------------------------- |
-| {{Glossary("Forbidden header name")}} | no                              |
+HTTP **`Cross-Origin-Resource-Policy`** 响应标头表示期望浏览器阻止对给定资源的 no-cors 跨源或跨站点请求。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/cross-origin-resource-policy/index.md:10]: “no-cors”为啥不翻译

Comment (T34-active) [inline files/zh-cn/web/http/headers/cross-origin-resource-policy/index.md:10]:
```suggestion
HTTP **`Cross-Origin-Resource-Policy`** 响应标头表示期望浏览器阻止对给定资源的非跨源资源共享或跨站点请求。
```

File: files/zh-cn/web/http/headers/cross-origin-resource-policy/index.md [hunk 2]
```diff
@@ -1,32 +1,43 @@
...
-Cross-Origin-Resource-Policy: same-site | same-origin
+```http
+Cross-Origin-Resource-Policy: same-site | same-origin | cross-origin
 ```
 
-## 案例
+## 示例
 
-下面的响应头会导致兼容该响应头的用户代理禁止跨域访问和跨域资源共享：
+以下响应标头将导致兼容的用户代理拒绝跨源 no-cors 请求：
```
Comment (T34-active) [inline files/zh-cn/web/http/headers/cross-origin-resource-policy/index.md:33]:
```suggestion
以下响应标头将导致兼容的用户代理拒绝非跨源资源共享请求：
```

File: files/zh-cn/web/http/headers/cross-origin-resource-policy/index.md [hunk 3]
```diff
@@ -1,32 +1,43 @@
...
 {{HTTPSidebar}}
 
-> **备注：** 由于[Chrome 浏览器中的一个 Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=952834), 设置 Cross-Origin-Resource-Policy（跨域资源策略）会使文件下载失败：当从设置了 CORP 请求头的资源服务器下载资源时，浏览器会阻止用户使用“保存”或“另存为”按钮将文件保存到本地。在决定生产环境中是否使用这一特性（CORP）之前需要慎重考虑。
-
-**`Cross-Origin-Resource-Policy`** 响应头会指示浏览器阻止对指定资源的无源跨域/跨站点请求。
-
-| Header type                           | {{Glossary("Response header")}} |
-| ------------------------------------- | ------------------------------- |
-| {{Glossary("Forbidden header name")}} | no                              |
+HTTP **`Cross-Origin-Resource-Policy`** 响应标头表示期望浏览器阻止对给定资源的非跨源资源共享或跨站点请求。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/cross-origin-resource-policy/index.md:10]:
```suggestion
HTTP **`Cross-Origin-Resource-Policy`** 响应标头表示期望浏览器阻止对给定资源在 no-cors 模式下的跨源或跨站点请求。
```

File: files/zh-cn/web/http/headers/cross-origin-resource-policy/index.md [hunk 4]
```diff
@@ -1,32 +1,43 @@
...
-Cross-Origin-Resource-Policy: same-site | same-origin
+```http
+Cross-Origin-Resource-Policy: same-site | same-origin | cross-origin
 ```
 
-## 案例
+## 示例
 
-下面的响应头会导致兼容该响应头的用户代理禁止跨域访问和跨域资源共享：
+以下响应标头将导致兼容的用户代理拒绝非跨源资源共享请求：
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/cross-origin-resource-policy/index.md:33]:
```suggestion
以下响应标头将导致兼容的用户代理拒绝 no-cors 模式下的跨源请求：
```

**Review by yin1999 — COMMENTED** [review]
参考 [Request.mode](https://developer.mozilla.org/zh-CN/docs/Web/API/Request/mode)，加上模式可能比较好。

**Review by yin1999 — APPROVED** [review]

**Pull request merged**